portfolio/src/main/webapp/style.css

```diff
@@ -123,3 +123,10 @@ body, html {
   text-align: center;
 }
 
+.comments-box{
+margin:0 auto; 
```

Comment:
this block looks like it's not indented; can you consistentize it with the above?

Reply:
yes, just did! sorry about that.